### PR TITLE
Adjust hero spacing on smaller screens

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Was messing around with @nloureiro in GH grooming and we made some small improvements to the PageHero's spacing on smaller screens, further changes will improve it on the DS implementation but this is a nice quick win.

## Related Issue

None

### CODE REVIEW

File: src/components/PageHero.tsx [hunk 1]
```diff
@@ -56,15 +56,17 @@ const PageHero: React.FC<IProps> = ({
...
           pl={{ base: 0, lg: 8 }}
-          mr={4}
+          mr={{ base: 0, lg: 4 }}
         >
           <Heading
             as="h1"
             textTransform="uppercase"
             fontSize="md"
             fontWeight="normal"
+            mt={{ base: "0px", lg: 8 }}
```
Comment (minimalsm) [inline src/components/PageHero.tsx:69]: For some reason, `base: 0` wasn't changing the base style at all here.

File: src/components/PageHero.tsx [hunk 2]
```diff
@@ -56,15 +56,17 @@ const PageHero: React.FC<IProps> = ({
       >
         <Box
           maxW={{ base: "full", lg: "container.sm" }}
-          py={{ base: 16, lg: 32 }}
+          pt={{ base: 0, lg: 32 }}
+          pb={{ base: 8, lg: 32 }}
           pl={{ base: 0, lg: 8 }}
-          mr={4}
+          mr={{ base: "0px", lg: 4 }}
```
Comment (minimalsm) [inline src/components/PageHero.tsx:62]: Same as below

image

**Comment by minimalsm** [comment]
> When we have images on top of the H1 should be some space.
> 

@nloureiro fixed [in this commit](https://github.com/ethereum/ethereum-org-website/pull/10379/commits/ce042390790d632a7071a6d72c649cdc154707c7).

Basically: 

- if the image is on top, add top padding, 
- if the image is below, add bottom padding.

**Review by nloureiro — APPROVED** [review]

**Comment by nloureiro** [comment]
All good on my side, approved :)

File: src/components/PageHero.tsx (hunk 1 above)
Comment (minimalsm) [reply src/components/PageHero.tsx:69]: Tried again local... same thing in all browsers 🤷‍♂️. Will update this PR and try in the preview deploy.

File: src/components/PageHero.tsx [hunk 3]
```diff
@@ -56,15 +56,17 @@ const PageHero: React.FC<IProps> = ({
       >
         <Box
           maxW={{ base: "full", lg: "container.sm" }}
-          py={{ base: 16, lg: 32 }}
+          pt={{ base: isReverse ? 0 : 8, lg: 32 }}
+          pb={{ base: isReverse ? 8 : 0, lg: 32 }}
           pl={{ base: 0, lg: 8 }}
-          mr={4}
+          mr={{ base: "0px", lg: 4 }}
```
Comment (minimalsm) [inline src/components/PageHero.tsx:62]:
```suggestion
          mr={{ base: 0, lg: 4 }}
```

**Review by corwintines — APPROVED** [review]

**Pull request merged**